bsoid_predict.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+from streamlit import caching
+
+from bsoid_app import predict
+from bsoid_app.bsoid_utilities import visuals
+from bsoid_app.bsoid_utilities.load_workspace import *
+
+def get_env_variable(var_name, default_value=None):
```

Comment:
You could probably move this to a `utils.py` and import it in both places to reduce duplicaion

Reply:
opting not to do this